### DISCUSSION TITLE
How to use Scale / Transform a GLTF node (with example patch)

### DISCUSSION THREAD

**Discussion by kikohs** [cables-gl/cables_docs#155] [Answers & Questions] [answered]
I'm trying to scale a mesh from my GLTF scene.
However if I apply a transform or ScaleXYZ node, the Y component doesn't behave as expected.
I tried to center the GLTF scene before but it doesn't behave correctly I think.

https://user-images.githubusercontent.com/1793640/154247035-a004bdde-8d00-40f4-86dc-685b3d006259.mp4

**Comment by pandrr** [comment, marked as answer]
the object alrady contains a transformation.

try to center the object in your 3d tool

or in cables: use aligngeometry, set everything to center, render this geometry with renderGeometry then the scaling should work as expected

**Comment by kikohs** [comment]
Awesome! Thanks. I'll make an example patch, could be helpful to others.

**Comment by kikohs** [comment]
Here is the example patch: 

https://cables.gl/p/eiFe_X
